app/controllers/katello/api/v2/host_errata_controller.rb

```diff
@@ -52,7 +57,11 @@ def show
     protected
 
     def index_relation
-      @host.content_facet.installable_errata(@environment, @content_view)
+      relation = []
+      if @host.content_facet
+        relation = @host.content_facet.installable_errata(@environment, @content_view)
+      end
+      relation
```

Comment:
This should really return some sort of active record relation even if its 'Erratum.where("1=0")'